### ISSUE TITLE
Add searches of people on the EA Donation Registry

### ISSUE THREAD

**Issue by tog22** [tog22/eahub#33]
These should go only on the appropriate EA Donation Registry pages, and be pre-filtered only for people who have past or planned donations shared. You could do separate searches for each or one combined search for people who have either shared - your judgement.

**Comment by tog22** [comment]
@haryshwaran - FYI, this might be easier to do when you move onto writing a new form element for past donations like we discussed verbally recently. It's likely (practically speaking) impossible for @wrsaunde to filter for people who have shared past donations as opposed to just having saved their 'profile_donations' profile and thus created a past donations table which only has the header row. ( @wrsaunde - it's acceptable for you to include these people, if you understand what I'm referring to, which I'd understand if you didn't!)

**Comment by william-r-s** [comment]
Right now, it's hard to do any kind of filtering on the donations tables (table fields aren't in the search index). You'd likely have to create another "hack" profile field, like I did in modules/ea_hub_misc/ea_hub_misc.module If you're planning to make modifications to how the field is stored soon, it might be best to do that first.

**Comment by tog22** [comment]
Yes - @haryshwaran can you make a github issue for this (so it's public to people like William), and then tell @wrsaunde here when you've made it and put the new donations table field into the search index.
